meson.build

```diff
@@ -49,8 +49,12 @@ endforeach
 _builtins = [
   ['__builtin_clz', '0', true],
   ['__builtin_types_compatible_p', 'int, int', true],
+  ['__builtin_uadd_overflow', '0U, 0U, (void*)0', false],
   ['__builtin_uaddl_overflow', '0UL, 0UL, (void*)0', false],
   ['__builtin_uaddll_overflow', '0ULL, 0ULL, (void*)0', false],
+  ['__builtin_umul_overflow', '0U, 0U, (void*)0', false],
+  ['__builtin_umull_overflow', '0UL, 0UL, (void*)0', false],
+  ['__builtin_umulll_overflow', '0ULL, 0ULL, (void*)0', false],
```

Comment:
Adding an inline comment as a suggestion for names: We could just stick with this and skip __builtin, i.e.
- `umul64_overflow` for unsigned multiplication of 64 bit values
- `umulsz_overflow` for size_t
And same goes for the additions
- `uadd64_overflow`
- `uaddsz_overflow`